src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -200,6 +200,8 @@ EditDataScriptFilePathNull = An output filename must be provided
 
 EditDataCommitInProgress = A commit task is in progress. Please wait for completion.
 
+EditDataComputedColumnPlaceholder = <TBD>
```

Comment:
I'm open for suggestions for what to use here. Though I think that using "NULL" isn't the best idea.